web/index.html

```diff
@@ -159,11 +170,14 @@ <h1 class="text-body-emphasis">UPK2ESPHome</h1>
             $("#logs").empty();
             $("#input").val("");
             $("#output").val("");
+            $("profileUrl").val("");
         } else {
             $.getJSON("https://tuya-cloudcutter.github.io/api/devices/" + this.value + ".json", function (data) {
+                slug = data.slug;
                 data = JSON.stringify(data, undefined, 2)
                 $("#input").val(data);
                 localStorage.setItem("input", data);
+                $("#profileUrl").val(document.location + "?profile=" + slug);
```

Comment:
Path would have `?profile` twice when visiting from a profile link.
```suggestion
                $("#profileUrl").val(document.location.origin + document.location.pathname + "?profile=" + slug);
```